third_party/move/move-model/bytecode/src/stackless_control_flow_graph.rs

```diff
@@ -251,6 +259,86 @@ impl StacklessControlFlowGraph {
     }
 }
 
+/// Iterator over blocks of a control flow graph in DFS order
+/// (always choosing the left-most child to visit first).
+pub struct DFSLeft<'a> {
+    successors: &'a BTreeMap<BlockId, Vec<BlockId>>,
```

Comment:
Looks like the prerequisite here is that `successors` have all blocks as keys, even if they don't have any successors. Perhaps document that here?

Reply:
done